objects/auth_factor.json

```diff
@@ -0,0 +1,55 @@
+{
+  "caption": "Authentication Factor",
+  "description": "An Authentication Factor object describes a category of methods used for identity verification in an authentication attempt.",
+  "name": "auth_factor",
+  "extends": "object",
+  "attributes": {
+    "factor_type": {
+      "requirement": "recommended",
+      "group": "primary"
+    },
+    "factor_type_id": {
+      "requirement": "required",
+      "group": "primary"
+    },
+    "device": {
+      "description": "Device used to complete an authentication request.",
+      "requirement": "recommended",
+      "group": "primary"
+    },
+    "provider": {
+      "description": "The name of provider for an authentication factor.",
+      "requirement": "recommended",
+      "group": "context"
+    },
+    "is_totp": {
+      "requirement": "recommended",
+      "group": "context"
+    },
+    "is_hotp": {
+      "requirement": "recommended",
+      "group": "context"
+    },
+    "security_questions": {
+      "requirement": "optional",
+      "group": "context"
+    },
+    "phone_number": {
```

Comment:
Pulling the JSON schema for the Auth class fails w/ a HTTP 500 on the Auth class -> http://18.219.208.151/schema/classes/authentication?profiles=

I think you need to define `phone_numer` in the `dictionary.json`, e.g.:

```json
    "phone_number": {
      "caption": "Phone Number",
      "description": "The number associated with the phone.",
      "type": "string_t"
    },
```

Reply:
Checkout out locally, 500 is gone

Reply:
Nice catch! Updated http://18.219.208.151/classes/authentication?extensions=splunk with the change.